### ISSUE TITLE
Erroneous warning for escaping commands with embedded slashes.

### ISSUE THREAD

**Issue by orbea** [koalaman/shellcheck#996]
#### For bugs
- Rule Id: SC1001
- My shellcheck version: online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/sh

un\alias -a

```

#### Here's what shellcheck currently says:

```
un\alias -a
  ^-- SC1001: This \a will be a regular 'a' in this context.
```

#### Here's what I wanted or expected to see:

If someone wanted to remove all aliases they would use `unalias -a`, however its best practice to also escape `unalias` to ensure its not an alias itself. The posix spec for `command` offers some examples along this line. 

http://pubs.opengroup.org/onlinepubs/9699919799/utilities/command.html

If the `\` is embedded shellcheck will print useless and erroneous error. It can be simply silenced prefacing the `\` like `\unalias -a` instead of `un\alias -a`, but it does not matter where the slash is placed to ensure its not an alias so it should not print a warning regardless.

**Comment by koalaman** [comment]
This is working as intended.

`\a` is a bell character in several other contexts, and in this example it does not help to disable aliases (because aliases are not enabled in scripts by default). Unlike a leading `\`, this is also not an established convention to avoid aliases when they *are* enabled.

Either write it in a canonical way (`command unalias` or `\unalias`) or disable the warning.
